src/test/java/de/retest/recheck/ui/descriptors/idproviders/RandomSuffixRetestIdProviderTest.java

```diff
@@ -0,0 +1,145 @@
+package de.retest.recheck.ui.descriptors.idproviders;
+
+import static de.retest.recheck.ui.Path.fromString;
+import static de.retest.recheck.ui.descriptors.IdentifyingAttributes.create;
+import static de.retest.recheck.ui.descriptors.IdentifyingAttributes.createList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Collection;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import de.retest.recheck.ui.descriptors.Attribute;
+import de.retest.recheck.ui.descriptors.IdentifyingAttributes;
+import de.retest.recheck.ui.descriptors.StringAttribute;
+import de.retest.recheck.ui.descriptors.SuffixAttribute;
+
+public class RandomSuffixRetestIdProviderTest {
```

Comment:
These are tests that do not actually precisely check the random part, moreover the `DefaultRetestIdProvider` (the same tests still pass for it). 

Does it make sense to provide a seed for the random generator, as you only generate 5 random characters and not a whole UUID?

Reply:
Again, not the focus of this PR.